djangobaseview/static/js/guestbook/view/GuestbookView.js

```diff
@@ -0,0 +1,105 @@
+define([
+	"dojo/_base/declare",
+	"dojo/_base/lang",
+	"dojo/_base/array",
+	"dojo/dom-construct",
+	"dojo/on",
+	"dojo/query",
+	"dijit/form/ValidationTextBox",
+	"dijit/form/Button",
+	"dijit/registry",
+	"./GreetingView",
+	"../GuestbookStore",
+	"./_ViewBaseMixin",
+	"dojo/text!./templates/Guestbook.html",
+], function(declare, lang, array, domConstruct, on, query, ValidationTextBox,
+			_WidgetsInTemplateMixin, registry, Greeting, GuestbookStore, _ViewBaseMixin, template) {
+
+		return declare([_ViewBaseMixin], {
```

Comment:
dòng code đầu tiên của class không nên thụt quá xa, sẽ làm cho toàn bộ code bị shift sang phải trong khi bên trái còn chỗ trống k0 dùng, thụt  1 tab là đủ

tương tự cho các chỗ khác.